common/PerfService.jsm

```diff
@@ -1,17 +1,18 @@
-/* globals Services */
 "use strict";
 
 /* istanbul ignore if */
 if (typeof ChromeUtils !== "undefined") {
-  ChromeUtils.import("resource://gre/modules/Services.jsm");
+  // eslint-disable-next-line no-var, vars-on-top
+  var {Services} = ChromeUtils.import("resource://gre/modules/Services.jsm");
```

Comment:
Maybe this instead?
```js
let Services;
if (typeof ChromeUtils !== "undefined") {
  ChromeUtils.import("resource://gre/modules/Services.jsm");
  Services = ChromeUtils.import("resource://gre/modules/Services").Services;
```

Reply:
That's what I had originally 😉 , but that just still has the local scoped Services blocking the global Services problem.

Reply:
Ah ok I see the problem. Leave it as is (might be worth commenting)

Reply:
Actually, I guess that was only a problem when I was doing double Proxy ?? I'll check again with `let Services`…

Reply:
I ended up adding a comment as to why `var` inside instead of `let` then also making import return the current `global`